AK/Variant.h

```diff
@@ -219,6 +219,8 @@ namespace AK {
 struct Empty {
 };
 
+constexpr bool operator==(Empty, Empty) { return true; }
```

Comment:
Why is this outside the struct itself?

Reply:
No good reason

Changed to Hendiadyoin1's suggestion for this